### PR TITLE
Fix the md hierarchy

### PR DESCRIPTION
**Pull request by liruifengv**

### CODE REVIEW

**Comment by liruifengv** [comment]
If this pr is approved, I will update the Chinese page also.

**Comment by Jutanium** [comment]
I'm thinking we should remove the `package.json#` prefix for each of the properties - we don't use this kind of prefix anywhere else in the docs. @sarah11918, thoughts?

**Comment by sarah11918** [comment]
Agreed, @Jutanium I would remove those because we don't use them in docs.

And just a quick note, @liruifengv - in future, please make sure to write a description for your PRs. Even for small or obvious PRs, it helps us make sure we know exactly what problem is being fixed so we know what we should be paying attention to, and also to help "categorize" the PR with the proper label! 🙌

File: src/pages/en/reference/publish-to-npm.mdx [hunk 1]
```diff
@@ -98,7 +98,7 @@ The `package.json` in the package directory includes all of the information rela
 }
 ```
 
-#### `description`
+#### `package.json#description`
```
Comment (Jutanium) [inline src/pages/en/reference/publish-to-npm.mdx:101]:
```suggestion
#### `description`
```

File: src/pages/en/reference/publish-to-npm.mdx [hunk 2]
```diff
@@ -108,7 +108,7 @@ A short description of your component used to help others know what it does.
 }
 ```
 
-#### `type`
+#### `package.json#type`
```
Comment (Jutanium) [inline src/pages/en/reference/publish-to-npm.mdx:111]:
```suggestion
#### `type`
```

File: src/pages/en/reference/publish-to-npm.mdx [hunk 3]
```diff
@@ -120,7 +120,7 @@ The module format used by Node.js and Astro to interpret your `index.js` files.
 
 We recommend using `"type": "module"` so that your `index.js` can be used as an entrypoint with `import` and `export`.
 
-### `package.json#homepage`
+#### `package.json#homepage`
```
Comment (Jutanium) [inline src/pages/en/reference/publish-to-npm.mdx:123]:
```suggestion
#### `homepage`
```

File: src/pages/en/reference/publish-to-npm.mdx [hunk 4]
```diff
@@ -148,7 +148,7 @@ The entry points of a package when imported by name.
 
 In this example, importing `my-component` would use `index.js`, while importing `my-component/astro` or `my-component/react` would use `MyAstroComponent.astro` or `MyReactComponent.jsx` respectively.
 
-#### `files`
+#### `package.json#files`
```
Comment (Jutanium) [inline src/pages/en/reference/publish-to-npm.mdx:151]:
```suggestion
#### `files`
```

File: src/pages/en/reference/publish-to-npm.mdx [hunk 5]
```diff
@@ -158,7 +158,7 @@ This is an optional optimization to exclude unnecessary files from the bundle sh
 }
 ```
 
-#### `keywords`
+#### `package.json#keywords`
```
Comment (Jutanium) [inline src/pages/en/reference/publish-to-npm.mdx:161]:
```suggestion
#### `keywords`
```

**Review by Jutanium — CHANGES_REQUESTED** [review]

**Comment by liruifengv** [comment]
@Jutanium @Jutanium Thank your help.

**Review by Jutanium — APPROVED** [review]

**Pull request merged**